app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -91,5 +91,34 @@
       </div>
       </fieldset>
     </div>
+    <!--advanced options-->
+    <div>
+      <a href=""
+         ng-click="showAdvancedOptions = true"
+         ng-show="!showAdvancedOptions">Use label selectors in the storage request </a>
+      <a href=""
+         ng-click="showAdvancedOptions = false"
+         ng-show="showAdvancedOptions">Hide label selectors in the storage request</a>
+
+
+      <div ng-show="showAdvancedOptions" class="form-group" >
+        <fieldset class="compute-resource" >
+          <label for="claim-name" >Label Selector</label>
+          <a href="https://docs.openshift.org/latest/install_config/persistent_storage/selector_label_binding.html">Learn more</a>
```

Comment:
Sorry missed this on the last review, but the link needs to go in app/scripts/constants.js so it can be customized for the different versions. Then use the `helpLink` directive to get the link.

https://github.com/openshift/origin-web-console/blob/master/app/scripts/constants.js#L11-L31

Here is an example:

https://github.com/openshift/origin-web-console/blob/e7a0c0a8f703d5429f70b78223abb31856a66670/app/views/set-limits.html#L21-L23

Make sure to have `target="_blank"` so it opens in a new window like our other help links.